### PR TITLE
add Frame support

### PR DESCRIPTION
**Pull request by goosewobbler**
Adding injected support for [Frame](https://frame.sh).

### CODE REVIEW

File: packages/connectkit/src/wallets/connectors/frame.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    logos: {
+      default: <Logos.Frame />,
+      transparent: <Logos.Frame />,
+      appIcon: <Logos.Frame />,
+      connectorButton: <Logos.Frame />,
+    },
+    logoBackground: '#fff',
+    scannable: false,
+    downloadUrls: {
+      download: 'https://connect.family.co/v0/download/frame',
```
[the remark this answers is not in the document]
Comment (lochie) [reply packages/connectkit/src/wallets/connectors/frame.tsx:21]: This `download` url is based off of our redirector which will detect platform/os and redirect the user to the relevant `downloadUrl`. This gets deployed when merged to `main`

**Comment by lochie** [comment]
Hey @goosewobbler, is this teal colour on the logo following brand guidelines for Frame? I haven't seen their logo used in this manner before.

CleanShot —05-01 at 12 12 39@2x

**Comment by goosewobbler** [comment]
> Hey @goosewobbler, is this teal colour on the logo following brand guidelines for Frame? I haven't seen their logo used in this manner before.
> 

Yes, we're trying a logo that will work both on dark and light themes.  This logo variant is currently used as the "connected" state of the companion extension.

**Comment by lochie** [comment]
> Yes, we're trying a logo that will work both on dark and light themes. This logo variant is currently used as the "connected" state of the companion extension.

@goosewobbler you could always use the theme variables to define the icon colour within the svg e.g `var(--ck-body-color)`

**Review by lochie — APPROVED** [review]

**Comment by lochie** [comment]
@goosewobbler sorry for the delay on this, this lgtm for the injected provider being detected, we're happy to merge this in.

regarding `eth-provider`, we have [a PR that will open this sort of functionality](https://github.com/family/connectkit/pull/184) on the way, just waiting for some reviews.

with regards to the wagmi connector, if that gets merged into wagmi we will actively set up support for it within our default setup as we want to keep 1:1 parity with whatever wagmi supports.

sorry again for the delay, and thanks for the contribution! 😊

**Pull request merged**